docs/api/examples/manage-env.mdx

```diff
@@ -61,6 +61,24 @@ Copying environments across projects or organizations is not allowed.
 Make sure you use an organization level api-key or project level api-key to copy an environment.
 :::
 
+:::note Supported objects
+Copy Environment currently copies the following objects:
+- [x] Resource
+    - [x] Resource Actions
+    - [x] Resource Attributes
+    - [x] Resource Roles
+    - [x] Resource Relations
+- [x] Roles
+    - [x] Role Permissions
+    - [ ] Role Derivations
+- [x] Condition Sets
+    - [x] User Sets
+    - [x] Resource Sets
+    - [x] Condition Sets Rules
+
+There is currently no support for ReBAC Role Derivations.
+:::
+
 ## Copy to a new environment
 
 In this example I'm coping `production` environment to a new environment called `production`.<br/>
```

Comment:
I'm coping `production` environment

this is a typo - replace with:
I'm coping an environment called `staging`